### DISCUSSION TITLE
Tailwind not working on install

### DISCUSSION THREAD

**Discussion by ChrisT303** [tailwindlabs/tailwindcss#9145] [Help] [answered]
I'm not sure what I could be doing wrong here. The tailwind classes will not apply no matter what I do. Can anyone possibly tell me what I'm doing wrong on the install? Thanks!

src/index.html:
```
<!doctype html>
<html>
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <link href="/dist/output.css" rel="stylesheet">
</head>
<body>
  <h1 class="text-6xl font-bold underline">
    Hello world!
  </h1>
</body>
</html>
```

config:
```
module.exports = {
  content: ["./src/**/*.{html,js}"],
  theme: {
    extend: {},
  },
  plugins: [],
};
```

json:
```
{
  "name": "cocktails",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "tailwindcss": "^3.1.8"
  }
}
```

**Comment by wongjn** [comment, marked as answer]
You need to generate your CSS file. Since you haven't mentioned that you are using an existing build system, we can use Tailwind CLI to do this:

```sh
npx tailwindcss -o src/dist/output.css
```

The path may need tweaking depending on how you are serving your `index.html`.

**Comment by ChrisT303** [comment]
Sorry for being unclear. I followed all the install instructions as per Tailwind, and for sure have an output.

**Reply by wongjn** [reply]
What do you get in your `output.css`?

**Reply by ChrisT303** [reply]
Actually, that did fix it! I just had to tweak the path just like you mentioned. Thanks so much!
